src/ServicePulse.Host/vue/src/components/configuration/HealthCheckNotifications_ConfigureEmail.vue

```diff
@@ -32,12 +32,20 @@ const fromIsValid = computed(() => {
   return from.value && emailRe.test(from.value) ? true : false;
 });
 const toIsValid = computed(() => {
-  return to.value && emailRe.test(to.value) ? true : false;
+    return to.value && validateMultipleEmailsSemicolonSeparated(to.value) ? true : false;
```

Comment:
```suggestion
  return to.value && validateMultipleEmailsSemicolonSeparated(to.value) ? true : false;
```

All other formats in this file only have 2 indents. Best not to change them